### ISSUE TITLE
how to use it

### ISSUE THREAD

**Issue by zuyangKIM** [ivanpfalcao/RGraphsForQliksense#1]
Can I just release it out into the extension folder? I don't know how to use this!

**Comment by ivanpfalcao** [comment]
Yes, you just need to download it and copy it to the extension folder. In Qliksense server you need to zip the extension folder and install it using QMC.

Sorry to don't respond the question earlier. I didn't recieve any notification.
